public/js/counties.js

```diff
@@ -479,7 +479,12 @@ window.onload = function() {
       $(".menu-button").click();
     }})
   Object.keys(countiesLocation).forEach(key => {
-  const c = new Option(countiesLocation[key].label, key)
+  const c = new Option(countiesLocation[key].label, key);
   countySelect.options.add(c);
-});
-};
+    if(c.value.toLowerCase() === "nairobi" ){
+      c.disabled = false
+    } else {
+      c.disabled = true
+    } 
```

Comment:
```suggestion
  // TODO(Brenda): We only have Nairobi data so we'll disable all other counties for now
  if(c.value !== "nairobi" ){
    c.disabled = true
  }
  countySelect.options.add(c);
```
1. county keys are already in lower case.
2. No need for `disabled: false`.